CMakeExt/CompilerFlags.cmake

```diff
@@ -36,7 +36,7 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES ".*Clang")
   # using Clang
   set (CXX_STD_FLAG "--std=c++11"
        CACHE STRING "C++ compiler std flag")
-  set (CXX_OMP_FLAG "-fopenmp")
+  set (CXX_OMP_FLAG "-fopenmp=libomp")
```

Comment:
Curious, clang-3.8 on my eval server required this. Will test again, perhaps it's just my custom configuration of llvm/clang.